### DISCUSSION TITLE
[求助]无法打开图片链接

### DISCUSSION THREAD

**Discussion by Attic-Leo** [share121/inter-knot#1044] [其他]
显示找不到 private-user-images.githubusercontent.com 的网页
我也试试[
![936](https://github.com/user-attachments/assets/33649348-c4d6-4105-9577-e0a979374cd6)
](url)

**Comment by Attic-Leo** [comment]
事实证明我这样似乎是可以的，但为什么有些还是看不到呢

**Comment by wugan-bot156** [comment]
对图片点击右键后选编辑图片，然后就可以保存了，而且保存的是原图。
